lib/howl/application.moon

```diff
@@ -311,7 +341,7 @@ class Application extends PropertyObject
   _restore_session: (window, restore_buffers) =>
     session = @settings\load_system 'session'
 
-    if session and session.version == 1
+    if session and session.version >= 1
```

Comment:
I wonder whether we should skip the version check completely for now, as we don't have any breaking incompatible changes yet. We could just check for the existence of `session.recently_closed` below. As it is, the version check seems wrong as it now accepts any future version of the session.